distribution/std-lib/Standard/src/Table/Data/Table.enso

```diff
@@ -166,6 +167,18 @@ type Table
             Json.from_pairs [['name', name], ['data', items]]
         Json.from_pairs [row_count, ['columns', cols]] . to_text
 
+    ## UNSTABLE
+       ADVANCED
+
+       Guides the visualization system to display the most suitable graphical
+       representation for this table.
+    default_visualization : Visualization.Id.Id
+    default_visualization =
+        cols = this.columns.map .name . map .to_lower_case
+        if cols.contains "latitude" && cols.contains "longitude" then Visualization.Id.geo_map else
+            if cols.contains "x" && cols.contains "y" then Visualization.Id.scatter_plot else
+                Visualization.Id.table
```

Comment:
I think we want to define visualization functionality in that library instead of on the types themselves. Such things as `to_default_visualization_data` should probably be moved at some point.

Reply:
I disagree – the default visualizations will be defined by library authors, so `Table` should do it for itself.